src/EnergyPlus/DataZoneEquipment.hh

```diff
@@ -164,6 +164,9 @@ namespace DataZoneEquipment {
         Num
     };
 
+    constexpr std::array<std::string_view, static_cast<int>(LoadDist::Num)> LoadDistNamesUC = {
+        "SEQUENTIALLOAD", "UNIFORMLOAD", "UNIFORMPLR", "SEQUENTIALUNIFORMPLR"};
```

Comment:
constexpr array, I plan on using `getEnumerationValue` (before @amirroth mentions it! :))

Reply:
It worked!